tests/test_live.py

```diff
@@ -0,0 +1,265 @@
+import io
+import os
+import random
+import sys
+import time
+from typing import List
+
+import pytest
+from rich.console import Console
+from rich.live import Live
+from rich.table import Table
+from rich.text import Text
+
+from tests.util import get_capture_text, set_capture_text
+
+
+def create_capture_console(*, width: int = 60, height: int = 80) -> Console:
+    return Console(
+        width=width,
+        height=height,
+        file=io.StringIO(),
+        force_terminal=True,
+        legacy_windows=False,
+        color_system=None,  # use no color system to reduce complexity of output
+    )
+
+
+def create_base_table() -> Table:
+    table = Table(title="test table", caption="table caption", expand=True)
+    table.add_column("foo", footer=Text("total"), no_wrap=True, overflow="ellipsis")
+    table.add_column("bar", justify="center")
+    table.add_column("baz", justify="right")
+
+    return table
+
+
+def check_output(output_file: str, output: str) -> None:
+    output = output.replace("\r", "")
+    if os.getenv("CAPTURE") is not None:  # adjust the correct output check
+        set_capture_text("live", output_file, output=output)
+
+    correct_output = get_capture_text("live", output_file).replace("\r", "")
+
+    if os.getenv("VERBOSE") is not None:
```

Comment:
Is this env some kind of standard? I generally leave print statements in tests, because pyTest will hide the output if the test passes anyway.

Reply:
Its not something standard. Just something I thought would be useful when testing locally (turns out it wasn't). Forgot to remove it.